src/common/InputContainer.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export function InputContainer({
+  children,
+  className,
+}: {
+  children: React.ReactNode;
+  className?: string;
+}) {
```

Comment:
```suggestion
}: PropsWithChildren<{className?: string}>) {
```

Small quality of life nit to avoid typing children all the time